src/Caching/perf/Microsoft.Extensions.Caching.Benchmarks/Microsoft.Extensions.Caching.Benchmarks.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Please follow the folder and project naming conventions this repo uses for microbenchmarks
e.g. https://github.com/dotnet/aspnetcore/blob/main/src/Middleware/OutputCaching/perf/Microbenchmarks/Microsoft.AspNetCore.OutputCaching.Microbenchmarks.csproj

Reply:
done: 1b12e1ab4d2339537b93222e79a92cfe5087f7ba